test/full_coverage_path_planner/test_full_coverage_path_planner.launch

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
```

Comment:
This file is practically identical to test/full_coverage_path_planner/test_full_coverage_path_planner.test

So I propose this file to be removed (since you can roslaunch `.test` files also and get exactly this).

Reply:
We received a comment of a user that he wanted a .launch file.

Reply:
That's okay, but then the `.test` file can simply be:
```
<launch>
   <include test.launch>
   <test> etc
</launch>
```
Et voila, minimal duplicate code.

Reply:
Checked!